### DISCUSSION TITLE
v13 beta 7 Remove direct access to the go2rtc API

### DISCUSSION THREAD

**Discussion by jakekeeys** [blakeblackshear/frigate#9179] [General]
just noticed this

```Changes since Beta 6
Remove direct access to the go2rtc API to mitigate potential remote code execution via the exec source
```
which i happened to be using to get an up to date single full resolution frame `/live/webrtc/api/frame.mp4?src=house`
which i'd then convert to jpeg

i guess this https://deploy-preview-6262--frigate-docs.netlify.app/integrations/api#get-apicamera_namerecordingsframe_timesnapshotpng 
is the best way to achieve this now?

i don't seem to be able to request a frame within the last 60 seconds is there a predictable buffer that needs to be considered?

the docs should probably mention the frametime is expected as an epoch

**Comment by NickM-27** [comment]
if you use the go2rtc api you can still map port 1984 and use it just the same as before
